Revision: hackerlab--lord--1.0--patch-37
Archive: lord@regexps.com--2002
Creator: Tom Lord <lord@regexps.com>
Date: Fri May 10 01:59:00 PDT 2002
Standard-date: 2002-05-10 08:59:00 GMT
Summary: very minor tweaks
Keywords: 
New-files: {arch}/hackerlab/hackerlab--lord/hackerlab--lord--1.0/lord@regexps.com--2002/patch-log/patch-37
  os/sys/mman.h
Modified-files: ChangeLog
  ChangeLog.d/lord@regexps.com--2002/ChangeLog.lord--1.0
  os/sys/stat.h rx-posix/match-regexp.c rx/super.c
New-patches: lord@regexps.com--2002/hackerlab--lord--1.0--patch-37


*** added files
 
    ./os/sys/mman.h

	Vector to system include for `mmap' and friends.
 

*** patched regular files
 
**** ./os/sys/stat.h

	Include the prereq (sys/types.h)
 
**** ./rx-posix/match-regexp.c
**** ./rx/super.c
 
 	Nits to start to shut up `gcc -O4 -Wall'.
 
